internal/pkg/agent/application/coordinator/coordinator_unit_test.go

```diff
@@ -1009,6 +1008,7 @@ inputs:
   - id: test-input
     type: filestream
     use_output: default
+    _runtime_experimental: otel
```

Comment:
```suggestion
    # This is temporary and it's gonna be removed/renamed before beats receivers are GA
    _runtime_experimental: otel
```
as a personal preference I think the simple `runtime` key was a better choice

Reply:
I like this one better, as it immediately communicates that the name is temporary. My primary concern was stopping users from depending on it.

Reply:
@pchila are we good to resolve this issue? 🙂

Reply:
Not blocking, resolving...